### PR TITLE
Fatima: string manipulation and formatting 

### PR DESCRIPTION
**Pull request by FatimaFarah93**
I've completed the steps to the exercise by creating String methods, printing out a statement that states that part of the username and name is the same through removing the dot and using the ignoreCase method. I've used String Builder and also printed out a statement using String formatting.

### CODE REVIEW

File: src/main/java/session5strings/StringManipulationAndFormattingFatima.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        String upperCaseName = name.toUpperCase();
+        boolean isTheUserNameEqualToName = emailAddress.replaceAll("\\.", " ")
+                .substring(0, 12).equalsIgnoreCase(name);
+
+        message.append(emailAddress).append(" ");
+        message.append(username).append(" ");
+        message.append(upperCaseName).append(" ");
+        message.append(isTheUserNameEqualToName);
+
+        System.out.printf("Email is is %s, Username is %s, the name is %s. Is username and name the same: %b"
```
Comment (owynowyn) [inline src/main/java/session5strings/StringManipulationAndFormattingFatima.java:35]: Here you could use the StringBuilder you created on line 23 as it looks like that is actually unused. I see how you've formatted it using the %s etc. replacements, an alternative could be to just add the nicely formatted texts to your `message.append` code. E.g.

> message.append("Email: ").append(emailAddress).append("\n");
> message.append("Username: ").append(username).append("\n");
> message.append("Name in Uppercase: ").append(upperCaseName).append("\n");
> message.append("Does the username match the name (ignoring case): ").append(isUserNameEqualToName);

File: src/main/java/session5strings/StringManipulationAndFormattingFatima.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package session5strings;
+
+import java.util.function.ToDoubleBiFunction;
```
Comment (owynowyn) [inline src/main/java/session5strings/StringManipulationAndFormattingFatima.java:3]: Unused import here that we could probably clear out

**Review by owynowyn — CHANGES_REQUESTED** [review]
Minor changes to formatting required, great work again though 🙏

**Comment by FatimaFarah93** [comment]
Pushed up the changes around printing the string builder and removing the unusued import @owynowyn

**Review by owynowyn — APPROVED** [review]
Looks good to me!

**Pull request merged**